spec/ttdrest/concerns/ad_group_spec.rb

```diff
@@ -642,7 +642,7 @@
             end
 
             %w(FrequencySettings SiteTargeting FoldTargeting
-            SupplyVendorAdjustments ContractTargeting SiteQualitySettings
+            SupplyVendorAdjustments SiteQualitySettings
```

Comment:
This was a failing spec upon pulling the repo, found late breaking megagon discussions in slack that put this value back in to use (but the spec was still asserting its absence)